docs/testing.md

```diff
@@ -2,19 +2,26 @@
 
 ## Quick start:
 
-Many of our browser tests have been rewritten as Python unit tests. The remaining rely on data that is not generated during testing, so please note many of these tests might fail if you do not have a production database locally. If you do, open a new Terminal window or tab and change to the project directory,
+To run browser tests, open a new Terminal window or tab and change to the project directory,
 then tell gulp to start the tests:
 
 ```sh
 gulp build
-gulp test:acceptance --suite=integration
-gulp test:acceptance --suite=content
+gulp test:acceptance
+```
+
+There are several options you can pass to run a particular suite of tests,
+to run a particular list of features,
+and/or to run it in "fast" mode:
 
+```sh
+gulp test:acceptance --suite=wagtail-admin
+gulp test:acceptance --specs=multi-select.feature
+tox -e acceptance-fast
```

Comment:
So you can do both but if you call tox it's:

` tox -e acceptance-fast specs=multi-select.feature`